server/routes/auth/auth.js

```diff
@@ -1,16 +1,51 @@
 const express = require('express');
 const router  = express.Router();
 const passport = require('passport');
+const db = require('../../db');
 
 router.get('/login', passport.authenticate('OAuth2', {scope: 'read:user'}, (req, res) => {
     const redirectURI = req.query.redirect || '/';
     res.redirect(redirectURI);
 }));
 
+const checkIsAdmin = async (id) => {
+  const { rows } = await db.query(`
+  SELECT COUNT(user_id)
```

Comment:
I was crying when I found out sql has no good `Boolean return type`.... 😢 ... I accept this solution 😃